src/services/Environment.service.ts

```diff
@@ -9,7 +9,8 @@ export const environmentPresets = {
   studio: 'studio',
   city: 'city',
   park: 'park',
-  lobby: 'lobby'
+  lobby: 'lobby',
+  soft: 'soft'
```

Comment:
This is the new HDRI wich lays in the AWS bucket at around 1.9mb.

Reply:
can we optimise maybe? as we did in the past? 1.9mb is quite a lot..

Reply:
If we cut the resolution in half down to 512x256 we can get to 530kb